hugo/content/tutorials/building_an_extension/index.md

```diff
@@ -32,7 +32,7 @@ npm i --save-dev esbuild
 
 ## Generate an Extension
 
-At this point we're ready to generate an extension. Ws need the VS Code Extension Manager (`vsce`) to do this, so make sure to download this from npm via `npm install -g vsce` (or install locally, as per your preference). Once you have that installed, you can invoke it like so from the root of your project.
+At this point we're ready to generate an extension. We need the VS Code Extension Manager (`vsce`) to do this, so make sure to download this from npm via `npm install -g vsce` (or install locally, as per your preference). Once you have that installed, you can invoke it like so from the root of your project.
```

Comment:
While `vsce` is still maintained, it is kind of deprecated. Microsoft recommends to use [`@vscode/vsce`](https://www.npmjs.com/package/@vscode/vsce) instead.

Reply:
Updated the install to reflect the more appropriate package location